ckanext/cfpb_extrafields/templates/package/resource_read.html

```diff
@@ -307,7 +307,7 @@ <h4 class="modal-title" id="myModalLabel">{{ _('Add a code snippet') }}</h4>
         <button id="btn_post_related_gist" class="btn btn-primary" type="button" data-module="post_related_gist"
                   data-module-dataset_id="{{ res.package_id }}"
                   data-module-resource_id="{{ res.id }}"
-                  data-module-email="{{c.userobj.email}}" {# accessing c. is bad practice! #}
+                  data-module-email="{{c.userobj.email}}" 
```

Comment:
@mattboehm Should we move this comment above the `<button>` tag and keep it as an html comment?  If it's something that should be changed, we should keep it as a TODO.